tmt/base.py

```diff
@@ -1694,7 +1684,6 @@ def environment(self) -> EnvironmentType:
         """ Return combined environment from plan data and command line """
         if self.my_run:
             combined = self._plan_environment.copy()
-            combined.update(self._environment)
```

Comment:
Is this removal correct? It looks like `combined` would lack the output of `environment_from_spec`, i.e. node's & CLI inputs.

Reply:
It's not and I did it on purpose, just trying to verify whether it would be caught by tests to see their code coverage. That's why it's a draft for now. :)